tests/MongoDB.Driver.Core.Tests/Specifications/initial-dns-seedlist-discovery/TestRunner.cs

```diff
@@ -35,22 +31,35 @@ public class TestRunner
     {
         [Theory]
         [ClassData(typeof(TestCaseFactory))]
-        public void RunTestDefinition(TestCase testCase)
+        public void RunTestDefinition(JsonDrivenTestCase testCase)
```

Comment:
Since we're working on this file anyway let's rename the class:

    public class InitialDnsSeedlistDiscoveryTestRunner

and the file too of course.

Reply:
done